packages/blocks/src/gauge.tsx

```diff
@@ -0,0 +1,32 @@
+import { PropsWithChildren } from "react";
+
+export type GaugeProps = PropsWithChildren<{
+  value: number;
+  min?: number;
+  max: number;
+}>;
+
+export function Gauge({ value, min = 0, max, children }: GaugeProps) {
```

Comment:
Not a great Gauge implementation, just threw this together in a few minutes - we should probably do this properly with visx at some point.

Reply:
agreed!